docker/jenkins/run_functional_tests.sh

```diff
@@ -36,4 +36,9 @@ do
     done;
 done;
 
-docker run -v `pwd`/results:/app/results --link bedrock-selenium-hub-${BUILD_NUMBER}:hub -e SELENIUM_HOST=hub -e BASE_URL=http://bedrock:8000 bedrock_functional_tests:$GIT_COMMIT
+docker run -v `pwd`/results:/app/results \
+--link bedrock-selenium-hub-${BUILD_NUMBER}:hub \
```

Comment:
Indent a couple of columns in to make it clear that this a continuation of the line above.